src/app/pages/features/common/common.html

```diff
@@ -31,13 +31,35 @@ <h2 id="Image" class="link clickable">
       <td><small><code>modal</code></small></td>
       <td>
         <div>
-          <i>Type: </i><code>ModalImage</code>
+          <i>Type: </i><a [routerLink]="['/features/modal-gallery']"><code>ModalImage</code></a>
         </div>
         <div>
           <p>Mandatory interface to configure the modal image.</p>
         </div>
       </td>
     </tr>
+    <tr>
+      <td><small><code>loading</code></small></td>
+      <td>
+        <div>
+          <i>Type: </i><code>'eager' | 'lazy'</code>
+        </div>
+        <div>
+          <p>Optional string to configure how the browser will load image. lazy by default.</p>
+        </div>
+      </td>
+    </tr>
+    <tr>
+      <td><small><code>fetchpriority</code></small></td>
+      <td>
+        <div>
+          <i>Type: </i><code>'high' | 'low' | 'auto'</code>
```

Comment:
the "optional" tag is missing:
`<span class="badge rounded-pill bg-secondary float-end">optional</span>`

Reply:
It's not really optional, it has a default value